usaspending_api/api_contracts/contracts/v2/references/toptier_agencies.md

```diff
@@ -27,6 +27,8 @@ This endpoint returns a list of toptier agencies, their budgetary resources, and
 + `agency_id` (required, number)
     `agency_id` is the unique USAspending.gov identifier for the agency. You will need to use this ID in other endpoints when requesting detailed information about this specific agency.
 + `agency_name` (required, string)
++ `url_name` (required, string)
```

Comment:
How do you feel about `agency_slug` instead?

Reply:
I think it's a weird, meaningless term, but also don't care; changed